app/decorators/resource_decorator.rb

```diff
@@ -27,6 +27,8 @@ def display_published_date
   end
 
   def display_doi
-    MintableDoiComponent.new(resource: resource_with_doi)
+    return if resource_with_doi.doi.blank?
+
+    MintingStatusDoiComponent.new(resource: resource_with_doi)
```

Comment:
No longer mint DOIs from the metadata table. This is why I made all these `-DoiComponent`s compose each other. So changing this is really simple